File: Chuongtrinh/DOANTOTNGHIEP/DOANTOTNGHIEP/Models/FileBTTL.cs
namespace DOANTOTNGHIEP.Models
{
    using System;
    using System.Collections.Generic;
    using System.ComponentModel.DataAnnotations;
    using System.ComponentModel.DataAnnotations.Schema;
    using System.Data.Entity.Spatial;

    [Table("FileBTTL")]
    public partial class FileBTTL
    {
        public long ID { get; set; }

        public long? MaBT { get; set; }

        public long? IDLibrary { get; set; }

        public virtual BaiTap BaiTap { get; set; }

        public virtual Library Library { get; set; }
    }
}

File: Chuongtrinh/cronjob/model/filebttl.go
package model

type FileBTTL struct {
	ID        *int64   `gorm:"primary_key;column:ID"`
	MaBT      int64    `gorm:"column:MaBT"`
	IDLibrary int64    `gorm:"column:IDLibrary"`
	Baitap    *Baitap  `gorm:"ForeignKey:ID;references:MaBT"`
	Library   *Library `gorm:"ForeignKey:ID;references:IDLibrary"`
}
